sql.txt

```diff
@@ -1,14 +1,28 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
 
+SELECT CustomerName, ContactName, Country FROM Customers ORDER BY Country LIMIT 9 OFFSET 10;
+
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
 
+SELECT * FROM Customers WHERE ContactName LIKE 'A%';
+
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
 
+SELECT * FROM OrderDetails WHERE ProductID=51 AND Quantity>10;
+
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
 
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Tofu", 6, 7, "40 - 100 g pkgs.", 23.25);
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Chais", 1, 1, "10 boxes x 20 bags", 18);
+INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Chang", 1, 1, "24 - 12 oz bottles", 10);
+
```

Comment:
Perfect. Another way to do this:

`INSERT INTO Products (ProductName, SupplierID, CategoryID, Unit, Price) VALUES ("Tofu", 6, 7, "40 - 100 g pkgs.", 23.25), ("Chais", 1, 1, "10 boxes x 20 bags", 18), ("Chang", 1, 1, "24 - 12 oz bottles", 10);`

Just to save you time and also using the method Don't Repeat Yourself (DRY).